src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -49,13 +50,28 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
 
             // If we need the host (if it exists), use it, otherwise target the console app directly.
-            string testConsoleAppArgs = "\"" + a.FullName + "\" " + t.FullName + " " + method.Name + " " + string.Join(" ", args);
+            var testConsoleAppArgs = new StringBuilder();
+            testConsoleAppArgs.Append(ExtraParameter);
```

Comment:
I added an option earlier that controls whether we quote each argument (on by default)

> This will also be interesting to see if Diagnostic.Process tokenizes differently on Unix systems.

It does look like the Unix builds are failing with TargetParameterCountException. I don't really know why - do you have any ideas?